### DISCUSSION TITLE
About to push a breaking change!

### DISCUSSION THREAD

**Discussion by mmcc-xx** [mmcc-xx/WhosAtMyFeeder#13] [Announcements]
I'm about to push new images. To use them you'll need to...
- delete your speciesid.db database from the data folder. All of your images are in there, so copy the database somewhere else if you want to keep them
- Add the URL of your frigate installation to the config file. See the sample config file in the repo

Lots of what I hope are improvements. No HA integration yet but I am adding sub-labels to Frigate.

**Comment by daernsinstantfortress** [comment]
Thanks - updated and looks to be working, although I need my fat pigeons to pay me a visit first so I can be sure...

Tip: For those using HAOS add-ons for their Frigate install and possibly running WhosAtMyFeeder on a different box, by default HA doesn't expose port 5000 for Frigate. You have to explicitly configure the Frigate add-on to expose it by going to the Frigate add-on -> Configuration tab and under network add port 5000 against the "Web Interface" config. Without this, it's only exposed internally and not on your wider network and thus won't work.

**Reply by mmcc-xx** [reply]
Thanks- I'll note that in the main readme
